msgpack/_packer.pyx

```diff
@@ -253,6 +254,15 @@ cdef class Packer(object):
                     raise ValueError("EXT data is too large")
                 ret = msgpack_pack_ext(&self.pk, longval, L)
                 ret = msgpack_pack_raw_body(&self.pk, rawval, L)
+            elif type(o) is TimestampType if strict_types else isinstance(o, TimestampType):
```

Comment:
No need to `isinstance`.  strict_types is just a backward compatibility option.
No need to care about supporting subclass of Timestamp.